### ISSUE TITLE
Include paragraph/stanza breaks in edit analysis

### ISSUE THREAD

**Issue by mdlincoln** [PghFrankenstein/fv-visualization#7]
Right now, I'm removing the flattened XML tags from the P1 files that indicate typographic formatting like paragraph and stanza breaks, so as to not have to deal with broken hierarchies created by the app vs. other element disjunctions. Eventually it'd be good to figure out even an approximate solution for this so that we can incorporate typographic change into the visualizations.

re: https://github.com/PghFrankenstein/fv-data/issues/8

**Comment by ebeshero** [comment]
Yes--definitely! I'm imagining a Python-ic solution where we check for missing close-tags and when those are absent, we supply them in a new batch of files for the purposes of the synoptic visuals.
